application/forms/Config/General/ApplicationConfigForm.php

```diff
@@ -88,6 +88,19 @@ public function createElements(array $formData)
             )
         );
 
+        $this->addElement(
+            'checkbox',
+            'security_use_csp',
+            [
+                'label'         => $this->translate('Enable strict content security policy'),
+                'description'   => $this->translate(
+                    'Set whether to to use strict content security policy (CSP).'
+                    . ' This setting prevents cross site scripting.'
```

Comment:
Unfortunately nothing really prevents `XSS` so I would write: `This setting helps to protect from cross-site scripting (XSS).`